packages/react-reconciler/src/__tests__/StrictEffectsMode-test.js

```diff
@@ -108,7 +120,11 @@ describe('StrictEffectsMode', () => {
       renderer.unmount();
     });
 
-    assertLog(['useLayoutEffect unmount', 'useEffect unmount']);
+    assertLog([
+      'useLayoutEffect unmount',
```

Comment:
I'm surprised this is the order of `unmount`s

Reply:
Yeah this looks off since the order for unmounts during update is different. But maybe this is caused by the missing double invocation during StrictMode?

Reply:
Is it missing? I couldn't tell if it was intentional that `useInsertionEffect` wasn't mounted twice.

Reply:
I'd try to make it mount twice first and maybe this fixes the wrong unmount order